src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -22,40 +22,6 @@ import scala.io.Source
 case class VCFHeaderInfo(sampleIds: Array[String], infoSignature: TStruct, vaSignature: TStruct, genotypeSignature: TStruct,
   filtersAttrs: VCFAttributes, infoAttrs: VCFAttributes, formatAttrs: VCFAttributes)
 
-object VCFLine {
-  def invalidRef(ref: String): Boolean = {
```

Comment:
I think this might be a problem unless we get rid of htsjdk completely.  Namely, there are VCFs with an N in the ref field (this is a character that appears in the h37 fasta) but is invalid by the VCF spec and htsjdk will throw an error.  I'm OK with deleting this, but we have to fix this before 0.2 final.

Reply:
Let's say yes, this would make me happy.